mobile/androidMediaSessionAPI.html

```diff
@@ -0,0 +1,232 @@
+<!DOCTYPE html>
+<!--
+*
+* Copyright (C) 2015, bitmovin GmbH, All Rights Reserved
+*
+* Created on: 2015-07-25 11:35:04
```

Comment:
This date is wrong, i don't know if we actually need it, maybe its better to just remove the autor + created